LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2011, Mozilla.
+Copyright (c) 2011, Mozilla Foundation.
```

Comment:
Curious how come this isn't the MPL 2.0?

Reply:
Ah I see it is/was BSD Licence originally. How come it's MoFo and not MoCo, though? (No rhyme scheme intended)

Reply:
This was a suggested fix from @willkg so not sure actually. I can look what we have in other repos. Or drop it back to how it was and leave it ambiguous.

Reply:
It should be Mozilla Foundation per the boilerplate we're supposed to use: https://mozilla-hub.atlassian.net/wiki/spaces/SLP/pages/27394064/Licensing+Runbook+Software+Licensing+And+Code+Contribution+Between+Mozilla+And+Other+Orgs+Projects